cockroachcloud/serverless-movr.md

```diff
@@ -0,0 +1,125 @@
+get-the-cluster-connection-string---
+title: Estimate Serverless Usage with MovR
+summary: Run the MovR sample application to estimate RU usage in CockroachDB Serverless.
+toc: true
+referral_id: docs_serverless_movr
+docs_area: get_started
+---
+
+This tutorial shows you how to initialize and run the MovR workload on a {{ site.data.products.serverless }} cluster, then use the Request Unit consumption data to estimate a sufficient budget for the MovR workload.
+
+## Overview
+
+MovR is a fictional company that offers users a platform for sharing vehicles, like scooters, bicycles, and skateboards, in select cities across the United States and Europe. The MovR team wants to use {{ site.data.products.serverless }} to run their high-growth, variable workload. They decide to test out a sample workload to see how it will perform and how much it will cost to run.
+
+## Before you begin
+
+- Make sure you have already [installed CockroachDB](../{{site.versions["stable"]}}/install-cockroachdb.html).
+- [Create a {{ site.data.products.serverless }} cluster](quickstart.html).
+
+## Step 1. Get the cluster connection string
+
+1. On the cluster's **Overview** page, click **Connect**.
```

Comment:
is there some kind of UI docs we can link to for "cluster's overview page" ?

Maybe this? https://www.cockroachlabs.com/docs/cockroachcloud/serverless-cluster-management.html#view-clusters-page

Reply:
yep good call, there's actually a Cluster Overview Page doc!